src/components/ContentHighlights/CurrentContentHighlightItemsHeader.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import { Button } from '@edx/paragon';
+import { useParams } from 'react-router-dom';
+import { TEST_COURSE_HIGHLIHTS_DATA } from './data/constants';
+
+const CurrentContentHighlightItemsHeader = () => {
+  const { highlightUUID } = useParams();
+  const [highlightTitle] = useState(TEST_COURSE_HIGHLIHTS_DATA.filter(
+    highlights => highlights.uuid === highlightUUID,
+  )[0].title);
+  return (
+    <>
+      { highlightTitle && (
+        <header>
```

Comment:
nit: im not sure if we should use the landmark `header` element here or not tbh. usually `header` is for navigation bar type headers? i might not fully understand when to use it though :)